source/validations.ts

```diff
@@ -20,14 +21,13 @@ class ValidationError extends Error {
 	 * describing the issue in detail.
 	 */
 	constructor(code: string, message: string) {
-		super(
-			`express-rate-limit: ${code} - ${message} See https://github.com/express-rate-limit/express-rate-limit/wiki/Error-Codes#${code.toLowerCase()} for more information on this error.`,
-		)
+		const url = `https://express-rate-limit.github.io/${code.toLowerCase()}`
```

Comment:
Oh, shoot, I wanted this to be uppercase so that it matched error.code. It was only downloaded because GitHub automatically downcases anchor tags :/

Reply:
```suggestion
		const url = `https://express-rate-limit.github.io/${code}/`
```

Might as well put in the trailing slash and spare the user one redirect, also.

Reply:
I did this fix and then updated express-rate-limit.github.io also so, for example, express-rate-limit.github.io/ERR_ERL_DOUBLE_COUNT works! And, if we ever need to change the URLs or give an error it's own page, it should be easy to do.

Reply:
Thanks!!